api/server.py

```
@@ -74,52 +45,62 @@ class HubSpotWebhookPayload(BaseModel):
-      "financial_health_signal": "Severely Negative",
-      "key_pain_point": "Permanent cessation of long steel production at Newcastle and related facilities, depletion of R1.683 billion IDC lifeline, failed exclusive negotiations for potential sale/restructuring, ongoing heavy losses, and heightened supply chain vulnerabilities amid structural challenges like high energy/logistics costs and import competition",
-      "sales_hook": "Avoid capital-heavy proposals. Focus on immediate crisis response services: short-term liquidity optimization, working capital advisory, retrenchment/restructuring consulting, employee transition support, and strategic advisory for asset divestment or operational wind-down to mitigate fallout from the Newcastle closure and broader long-steel shutdown"
+        "source": "Live Search (Simulated)",
+        "company": company,
+        "status": "Integration Pending - Configure Perplexity/Google API",
+        "timestamp": time.time()
     }
 
- feat/modernize-python-stack-2026-3829493454699415671
 @app.post("/webhook/hubspot")
 async def handle_hubspot_webhook(payload: HubSpotWebhookPayload):
```

Ruff (0.14.10)

Unused function argument: `payload`

(ARG001)